pkg/transform/oauth/keystone_test.go

```diff
@@ -17,7 +21,30 @@ func TestTransformMasterConfigKeystone(t *testing.T) {
 
 	file := "testdata/keystone-test-master-config.yaml"
 	content, _ := ioutil.ReadFile(file)
-	masterV3 := ocp3.MasterDecode(content)
+	serializer := k8sjson.NewYAMLSerializer(k8sjson.DefaultMetaFactory, scheme.Scheme, scheme.Scheme)
+	var masterV3 configv1.MasterConfig
+	_, _, _ = serializer.Decode(content, nil, &masterV3)
```

Comment:
Looks like this is done in every unit test for the most part. So if it's intentional ignore.

Reply:
It is intentional. There's probably some work to be done to clean this up. 

MasterDecode and Extract don't exactly map 1 to 1. This gets the tests working, but I don't think it's ideal.